lib/steam_valve/steam_user.rb

```diff
@@ -5,4 +5,10 @@ def self.game_list(steam_id)
     @api_endpoint_url = "http://api.steampowered.com/IPlayerService/GetOwnedGames/v0001/?key=#{SteamAPIKey.api_key}&steamid=#{steam_id}&include_appinfo=1&format=json"
     HTTParty.get(@api_endpoint_url).to_hash
   end
-end
\ No newline at end of file
+
+  def self.current_players(game_id)
+    @api_endpoint_url =  "https://api.steampowered.com/ISteamUserStats/GetNumberOfCurrentPlayers/v1/?appid=#{game_id}"
```

Comment:
You could probably start extracting parts of this URL and the actual API call out as you use it more.